packages/rspack/src/config/defaults.ts

```diff
@@ -893,9 +893,10 @@ const applyOptimizationDefaults = (
 	D(optimization, "removeAvailableModules", true);
 	D(optimization, "removeEmptyChunks", true);
 	D(optimization, "mergeDuplicateChunks", true);
-	F(optimization, "moduleIds", (): "named" | "deterministic" => {
+	F(optimization, "moduleIds", (): "named" | "natural" | "deterministic" => {
 		if (production) return "deterministic";
-		return "named";
+		if (development) return "named";
+		return "natural";
```

Comment:
It is a breaking change, should implement "moduleIds=natural" without changing the default value. We will change it in next minor version.

Reply:
The changes to defaults have been reverted. Also added a config test. PTAL~